docs/content/ingestion/ingestion-spec.md

```diff
@@ -208,6 +208,7 @@ handle all formatting decisions on their own, without using the ParseSpec.
 |-------|------|-------------|----------|
 | column | String | The column of the timestamp. | yes |
 | format | String | iso, posix, millis, micro, nano, auto or any [Joda time](http://joda-time.sourceforge.net/apidocs/org/joda/time/format/DateTimeFormat.html) format. | no (default == 'auto' |
+| timeZone | String | Specify timeZone of the timestamp. e.g. "timestampSpec": {"format": "yyyy-MM-dd HH:mm:ss","column": "logTime","timestamp": "America/Los_Angeles"} | no (default == 'UTC') |
```

Comment:
Suggest removing the inline json example, and replacing with the link to list all supported values for `timeZone` parameter, maybe re use the [joda supported timezones list](http://joda-time.sourceforge.net/timezones.html) that is used by granularities documentation

Reply:
it's a good suggestion, I've removed the example and added a timezone link